public/components/event_analytics/explorer/__tests__/data_grid.test.tsx

```diff
@@ -28,21 +37,44 @@ describe('Datagrid component', () => {
       [SELECTED_FIELDS]: [],
       [UNSELECTED_FIELDS]: [],
       [AVAILABLE_FIELDS]: SIDEBAR_AVAILABLE_FIELDS,
-      [QUERIED_FIELDS]: QUERY_FIELDS
+      [QUERIED_FIELDS]: QUERY_FIELDS,
     };
-    
+
+    httpClientMock.get = jest.fn(() =>
+      Promise.resolve((sampleEmptyPanel as unknown) as HttpResponse)
+    );
+
+    const http = httpClientMock;
+    const pplService = new PPLService(httpClientMock);
+    const store = createStore(rootReducer, applyMiddleware(thunk));
+
     const wrapper = mount(
-      <DataGrid
-        rows={DATA_GRID_ROWS}
-        rowsAll={[]}
-        explorerFields={explorerFields}
-      />
+      <Provider store={store}>
+        <DataGrid
+          http={http}
+          pplService={pplService}
+          rows={DATA_GRID_ROWS}
+          rowsAll={[]}
+          explorerFields={explorerFields}
+          timeStampField={'timestamp'}
+          rawQuery={
+            "source = opensearch_dashboards_sample_data_logs | where match(request,'filebeat')"
```

Comment:
Minor: Might want to move this to test constants: https://github.com/opensearch-project/dashboards-observability/blob/main/test/event_analytics_constants.ts

Reply:
Moved